ui/imports/shared/status/StatusStickersPopup.qml

```diff
@@ -97,6 +96,16 @@ Popup {
                 stickersContainer.visible = true
             }
 
+            Connections {
+                target: root.store.stickersModuleInst
+                onStickerPackInstalled: {
```

Comment:
`function onStickerPackInstalled`, to make it compilable with Qt >= 5.15

Reply:
done